socorro/external/postgresql/adi.py

```diff
@@ -23,14 +23,16 @@ def get(self, **kwargs):
             ('start_date', yesterday, 'date'),
             ('end_date', tomorrow, 'date'),
             ('product', '', 'str'),
-            ('version', '', 'str'),
+            ('versions', [], 'list'),
+            ('platforms', [], 'list'),
```

Comment:
I think we discussed this elsewhere. The fact that SuperSearch allows a value to be a string or a list is an inherited thing from out of ES. If you aks me, it's kinda strange to not call it "versions", "products", "platforms" everywhere. We often have code that looks something like this:
```python
def find_things(platforms):
    if isinstance(platforms, basestring):
        # allows people to be lazy and type `find_things('osx')`
         platforms = [platforms]
```

I think the `s` adds value to the code-reader and implementor. You can immediately see that the implementation allows for it to be a multi-value. Like `FirstDateSignatures.get(signatures=[....])`. That tells, just from reading that that you can use a list. 

If we don't have the `s`, people might think that you can send either a value or a list of values and it might break horribly if it doesn't work. 

In the world of PG, which we're never going to get rid of, I think it makes sense to have humane sounding arguments.